docs/Module-Security.md

```
@@ -46,6 +46,24 @@ Terasology relies on [Gestalt Module Sandboxing](https://github.com/MovingBlocks
 
 * [o.t.engine.core.module.ExternalApiWhitelist](https://github.com/MovingBlocks/Terasology/blob/develop/engine/src/main/java/org/terasology/engine/core/module/ExternalApiWhitelist.java) defines a hardcoded list of allowable packages and classes.
 
+### What counts as permitted
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)